docs/topics/auth/default.txt

```diff
@@ -354,23 +354,34 @@ If you have an authenticated user you want to attach to the current session
                 # Return an 'invalid login' error message.
                 ...
 
-.. admonition:: Calling ``authenticate()`` first
+    .. versionchanged:: 1.10
 
-    When you're manually logging a user in, you *must* successfully authenticate
-    the user with :func:`~django.contrib.auth.authenticate()` before you call
-    :func:`~django.contrib.auth.login()`.
-    :func:`~django.contrib.auth.authenticate()`
-    sets an attribute on the :class:`~django.contrib.auth.models.User` noting
-    which authentication backend successfully authenticated that user (see the
-    :ref:`backends documentation <authentication-backends>` for details), and
-    this information is needed later during the login process. An error will be
-    raised if you try to login a user object retrieved from the database
-    directly.
+        Added the optional ``backend`` argument. Read the box bellow for more details.
+
+
+.. admonition:: Selecting the :ref:`authentication backend <authentication-backends>`
```

Comment:
I don't really care for the use of the admonition box here as it's describing the functionality of the login function, not some related topic which is often what boxes are used for.